.github/workflows/build_images.yaml

```diff
@@ -40,15 +40,8 @@ jobs:
       uses: docker/login-action@v3
       with:
         registry: harbor.stfc.ac.uk
-        username: ${{ secrets.HARBOR_USERNAME }}
-        password: ${{ secrets.HARBOR_TOKEN }}
-    - name: Build & push to prod
-      if: ${{ github.ref == 'refs/heads/master' }}
-      uses: docker/build-push-action@v3
-      with:
-        push: true
-        context: "{{defaultContext}}:${{ matrix.image }}"
-        tags: "harbor.stfc.ac.uk/stfc-cloud/${{ matrix.image }}:latest"
+        username: ${{ secrets.STAGING_HARBOR_USERNAME }}
+        password: ${{ secrets.STAGING_HARBOR_TOKEN }}
```

Comment:
Copying my comment from Slack:

Could we just wrap the different logins using an if:
```
if: ${{ github.ref == 'refs/heads/master' }}
```